api/src/test/java/org/openmrs/module/sync2/api/service/impl/atomfeed/LocalAtomfeedFeedReaderTest.java

```diff
@@ -0,0 +1,9 @@
+package org.openmrs.module.sync2.api.service.impl.atomfeed;
+
+
+public class LocalAtomfeedFeedReaderTest {
+	
+public void LocalAtomfeedFeedReader_shouldNotSyncIfPushIsDisabled() {
+		
```

Comment:
@reagan-meant What are we testing here?

Reply:
havent written tests for this yet..Still focusing on how to instatiate the parentfeedreader in the other test class...